### DISCUSSION TITLE
Preset for AB

### DISCUSSION THREAD

**Discussion by MrVSV** [gillett-hernandez/Foreman2#2] [General]
![00002319](https://user-images.githubusercontent.com/99773044/194744522-9a6372dd-8e4c-4328-a6b3-5d9dfcf4b98f.png)
[Foreman.zip](https://github.com/gillett-hernandez/Foreman2/files/9741025/Foreman.zip)

**Comment by gillett-hernandez** [comment]
hi, it looks like the issue is caused by the mod "Mapshot". if you disable it in factorio and then retry the export, it should work.

**Reply by MrVSV** [reply]
Thanks a lot
I'll try
